src/parser/csql_grammar.y

```diff
@@ -530,6 +529,14 @@ static PT_NODE * pt_create_date_value (PARSER_CONTEXT *parser,
 static PT_NODE * pt_create_json_value (PARSER_CONTEXT *parser,
 				       const char *str);
 static void pt_jt_append_column_or_nested_node (PT_NODE * jt_node, PT_NODE * jt_col_or_nested);
+
+typedef struct {
+        char* pUrl;
+        char* pUser;
+        char* pPwd;
+} SDbLinkConnInfo;
```

Comment:
Is this `SDbLinkConnInfo` struct necessary?

Reply:
Instead of using a structure, we can implement the same functionality by changing the parameters of the pt_ct_check_fill_connection_info() function.

Do you have a reference page for naming convention?